### PR TITLE
Concluding and Notes

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
This is only notes A-G with the italicization added back in....Jared wanted me to do smaller chunks to make sure my italicizing doesn't disappear again. So I'm doing like 7 notes at a time. Seems to be working okay so far!

File: diary-alexander-jaffray/modernized/37-concluding-observations.adoc [hunk 1]
```diff
@@ -1,9 +1,7 @@
 == Concluding Observations
 
-On the state of the church at large,
-with reference to the successive stages of her reformation,
-and occasioned by a view of that share and interest,
-which the Society of Friends have ever taken therein.
+[.chapter-synopsis]
+* On the state of the church at large, with reference to the successive stages of her reformation, and occasioned by a view of that share and interest, which the Society of Friends have ever taken therein
```
Comment (Henderjay) [inline diary-alexander-jaffray/modernized/37-concluding-observations.adoc:4]: This chapter-synopsis looks good. But looks like you forgot to do it on the original too. :)

File: diary-alexander-jaffray/modernized/38-notes-memoirs.adoc [hunk 2]
```diff
@@ -795,10 +834,17 @@ And do not think the time long, for all time is in the Father`'s hand, his power
...
 
+--
+
+[.centered]
 === Note P.
 
+[.embedded-content-document.address]
+--
+
 A Seasonable Warning and serious exhortation to,
```
Comment (Henderjay) [inline diary-alexander-jaffray/modernized/38-notes-memoirs.adoc]: This little chunk, starting with "A Seasonable Warning..." etc I think would look better as a blurb heading, or something like that. Both here and in the original. 👍🏻

**Review by Henderjay — CHANGES_REQUESTED** [review]
Hey Maija, just a couple things to change on this one. The Concluding Observations needs the chapter synopsis on the original version. And I thought that Note P. (in both the original and modernized) needs a blurb heading. 👍🏻

**Comment by Henderjay** [comment]
Ok, this all looked good. Merged!

**Pull request merged**